.github/workflows/contracts.yml

```diff
@@ -0,0 +1,54 @@
+name: Solidity
+
+#TODO: extend the conditions once workflow gets tested together with other workflows
+on:
+  push:
+    branches:
+      # TODO: Run on master after we're fully migrated from Circle CI
+      # - master
+      - "rfc-18/**"
+    paths:
+      - "solidity/**"
+      - ".github/workflows/contracts.yml"
+  pull_request:
+    branches:
+      # TODO: Run on all branches after we're fully migrated from Circle CI
+      - "rfc-18/**"
+    paths:
+      - "solidity/**"
+      - ".github/workflows/contracts.yml"
+
+
```

Comment:
Let's remove one of the duplicated empty lines.

Reply:
I've spotted it later and fixed that while working on #755, so it will be fixed once that PR gets accepted.